components/ServiceCard/index.js

```diff
@@ -1,5 +1,8 @@
-import Image from "next/image";
+import StyledImage from "../Layout/StyledImage";
 import useSWR from "swr";
+import H2 from "../Layout/H2";
```

Comment:
We recommend you change the name of "H2" to something else because it is hard to read the difference between "h2" and "H2". A styled component should have an explanatory name.  For example, "StyledHeader".